src/main/java/io/cdap/plugin/servicenow/sink/transform/ServiceNowTransformer.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.servicenow.sink.transform;
+
+import com.google.gson.JsonObject;
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.api.data.format.UnexpectedFormatException;
+import io.cdap.cdap.api.data.schema.Schema;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
+
+/**
+ *  Transforms structured record to JSON object
+ */
+public class ServiceNowTransformer {
+  private static final Logger LOG = LoggerFactory.getLogger(ServiceNowTransformer.class);
+  private static final int DEFAULT_SCALE = 2;
+  private static final String DATE_PATTERN = "yyyy-MM-dd";
+  private static final String DATE_TIME_PATTERN = "yyyy-MM-dd'T'HH:mm:ss";
+
+  @Nullable
+  public JsonObject transform(@Nullable StructuredRecord record) {
```

Comment:
when can record be null?

Reply:
Removed `@Nullable` as it cannot be null. a1691d4500f8670c38f28b341322dac56d37f279